docs/sphinx_api_docs_source/data_context.md

```diff
@@ -0,0 +1,8 @@
+# DataContext
```

Comment:
This is an example class source document. The file supports markdown because of the myst-parser but the directive needs to be in rst format. Typically we should be able to copy paste this format and modify just the class name in the directive and title.